### DISCUSSION TITLE
Documentation in Brazilian Portuguese

### DISCUSSION THREAD

**Discussion by cpereiraweb** [filamentphp/filament#160] [Ideas]
@danharrin, unfortunately many Brazilian developers are not fluent in English. Alongside this, PHP and Laravel are inclusive tools that allow many to support themselves here. Is there already a repository for Filament documents?
A confession of thanks: that PR was my overcoming of the imposter syndrome. I need to give back to the community how much I have already received. If you agree, I would like to work on translating the Filament documentation.
Perhaps this will allow more developers here in Brazil to feel motivated to try Laravel and change their own lives, just as I, at 52, am changing mine.

**Comment by ryangjchandler** [comment]
> @danharrin, unfortunately many Brazilian developers are not fluent in English. Alongside this, PHP and Laravel are inclusive tools that allow many to support themselves here. Is there already a repository for Filament documents?
> A confession of thanks: that PR was my overcoming of the imposter syndrome. I need to give back to the community how much I have already received. If you agree, I would like to work on translating the Filament documentation.
> Perhaps this will allow more developers here in Brazil to feel motivated to try Laravel and change their own lives, just as I, at 52, am changing mine.

@cpereiraweb Please feel free to open a pull request with some translations for the rest of the application!

**Comment by cpereiraweb** [comment]
@ryangjchandler, it was only after I wrote here that I found the 'docs' repository. Forgive me for that. I made a fork of it and I'm going to start working on the translation. Thanks for the support.
